### PR TITLE
[excel] (ExcelApi 1.12) Adding content for the ExcelApi 1.12 requirement set

### PR DESCRIPTION
**Pull request by AlexJerabek**
This PR:
- Creates a page for the 1.12 requirement set.
- Updates the preview req set page to reflect the promotion of 1.12.
- Updates the ExcelApiOnline req set page.
- Removes preview tags from associated article sections (including CF 1.3).

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[bf5b73f](https://github.com/OfficeDev/office-js-docs-pr/commits/bf5b73f4aa8d6d5c8b1ab5a27db8fc6086fd5070)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-errors.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/excel/custom-functions-errors.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-errors?branch=pr-en-us-2180) |
[docs/excel/excel-add-ins-events.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/excel/excel-add-ins-events.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/excel-add-ins-events?branch=pr-en-us-2180) |
[docs/excel/excel-add-ins-ranges-advanced.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/excel/excel-add-ins-ranges-advanced.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/excel-add-ins-ranges-advanced?branch=pr-en-us-2180) |
[docs/excel/excel-add-ins-workbooks.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/excel/excel-add-ins-workbooks.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/excel-add-ins-workbooks?branch=pr-en-us-2180) |
[docs/reference/requirement-sets/excel-api-1-12-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/reference/requirement-sets/excel-api-1-12-requirement-set.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-api-1-12-requirement-set?branch=pr-en-us-2180) |
[docs/reference/requirement-sets/excel-api-online-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/reference/requirement-sets/excel-api-online-requirement-set.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-api-online-requirement-set?branch=pr-en-us-2180) |
[docs/reference/requirement-sets/excel-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/reference/requirement-sets/excel-api-requirement-sets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets?branch=pr-en-us-2180) |
[docs/reference/requirement-sets/excel-preview-apis.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/reference/requirement-sets/excel-preview-apis.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-preview-apis?branch=pr-en-us-2180) |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-errors?branch=pr-en-us-2180) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C9%5C15%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202009151851519501-2180%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=RJVXioIEdUZRlW2T4Nv3YazBPIMgUZHZ3ilPnoAeHes%3D&st=2020-09-15T18%3A47%3A32Z&se=2020-10-16T18%3A52%3A32Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Review by alison-mk — APPROVED** [review]
Looks good to me!

File: docs/excel/custom-functions-errors.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
-ms.date: 05/06/2020
+ms.date: 09/15/2020
 description: 'Handle and return errors like #NULL! from your custom function'
```
Comment (ElizabethSamuel-MSFT) [inline docs/excel/custom-functions-errors.md:3]:
```suggestion
description: 'Handle and return errors like #NULL! from your custom function.'
```

File: docs/reference/requirement-sets/excel-api-1-12-requirement-set.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
+---
+title: Excel JavaScript API requirement set 1.12
+description: 'Details about the ExcelApi 1.12 requirement set'
```
Comment (ElizabethSamuel-MSFT) [inline docs/reference/requirement-sets/excel-api-1-12-requirement-set.md:3]:
```suggestion
description: 'Details about the ExcelApi 1.12 requirement set.'
```

File: docs/reference/requirement-sets/excel-api-online-requirement-set.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Excel JavaScript API online-only requirement set
 description: 'Details about the ExcelApiOnline requirement set'
```
Comment (ElizabethSamuel-MSFT) [inline docs/reference/requirement-sets/excel-api-online-requirement-set.md:3]:
```suggestion
description: 'Details about the ExcelApiOnline requirement set.'
```

File: docs/reference/requirement-sets/excel-preview-apis.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Excel JavaScript preview APIs
 description: 'Details about upcoming Excel JavaScript APIs'
```
Comment (ElizabethSamuel-MSFT) [inline docs/reference/requirement-sets/excel-preview-apis.md:3]:
```suggestion
description: 'Details about upcoming Excel JavaScript APIs.'
```

**Review by ElizabethSamuel-MSFT — COMMENTED** [review]
Nit: I believe the description metadata property should be a sentence and therefore end with a period.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[f304c10](https://github.com/OfficeDev/office-js-docs-pr/commits/f304c10d913dcb7ca3df0d02f3b29ba88579dff5)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-errors.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/excel/custom-functions-errors.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-errors?branch=pr-en-us-2180) |
[docs/excel/excel-add-ins-events.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/excel/excel-add-ins-events.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/excel-add-ins-events?branch=pr-en-us-2180) |
[docs/excel/excel-add-ins-ranges-advanced.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/excel/excel-add-ins-ranges-advanced.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/excel-add-ins-ranges-advanced?branch=pr-en-us-2180) |
[docs/excel/excel-add-ins-workbooks.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/excel/excel-add-ins-workbooks.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/excel-add-ins-workbooks?branch=pr-en-us-2180) |
[docs/reference/requirement-sets/excel-api-1-12-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/reference/requirement-sets/excel-api-1-12-requirement-set.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-api-1-12-requirement-set?branch=pr-en-us-2180) |
[docs/reference/requirement-sets/excel-api-online-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/reference/requirement-sets/excel-api-online-requirement-set.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-api-online-requirement-set?branch=pr-en-us-2180) |
[docs/reference/requirement-sets/excel-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/reference/requirement-sets/excel-api-requirement-sets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets?branch=pr-en-us-2180) |
[docs/reference/requirement-sets/excel-preview-apis.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/reference/requirement-sets/excel-preview-apis.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-preview-apis?branch=pr-en-us-2180) |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-errors?branch=pr-en-us-2180) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C9%5C15%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202009152120445522-2180%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=qw6kr9ZxyncReaWq6G8snL7Z5o324uy7Qa4Y5UPleUU%3D&st=2020-09-15T21%3A16%3A18Z&se=2020-10-16T21%3A21%3A18Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by AlexJerabek** [comment]
Thanks for the feedback @ElizabethSamuel-MSFT. I think I addressed your concerns (by taking every one of your suggestions, lol)

File: docs/reference/requirement-sets/excel-api-online-requirement-set.md [hunk 5]
```diff
@@ -35,20 +35,10 @@ Once the API is in a cross-platform requirement set, you should remove or edit t
...
-||[getItem(key: string)](/javascript/api/excel/excel.pivottablescopedcollection#getitem-key-)|Gets a PivotTable by name.|
-||[getItemOrNullObject(name: string)](/javascript/api/excel/excel.pivottablescopedcollection#getitemornullobject-name-)|Gets a PivotTable by name. If the PivotTable does not exist, will return a null object.|
-||[items](/javascript/api/excel/excel.pivottablescopedcollection#items)|Gets the loaded child items in this collection.|
-|[Range](/javascript/api/excel/excel.range)|[getPivotTables(fullyContained?: boolean)](/javascript/api/excel/excel.range#getpivottables-fullycontained-)|Gets a scoped collection of PivotTables that overlap with the range.|
+There are currently no APIs in the `ExcelApiOnline` requirement set. All the APIs that were previously a part of this set have graduated to a numbered requirement set and are available across all platforms.
 
 ## See also
 
 - [Excel JavaScript API Reference Documentation](/javascript/api/excel?view=excel-js-online&preserve-view=true)
 - [Excel JavaScript preview APIs](./excel-preview-apis.md)
```
Comment (ElizabethSamuel-MSFT) [inline docs/reference/requirement-sets/excel-api-online-requirement-set.md:43]: If they're in the same folder as the current file, you don't need `./`, right?
```suggestion
- [Excel JavaScript preview APIs](excel-preview-apis.md)
```

File: docs/reference/requirement-sets/excel-api-online-requirement-set.md [hunk 6]
```diff
@@ -35,20 +35,10 @@ Once the API is in a cross-platform requirement set, you should remove or edit t
...
-- [Excel JavaScript API requirement sets](./excel-api-requirement-sets.md)
\ No newline at end of file
+- [Excel JavaScript API requirement sets](./excel-api-requirement-sets.md)
```
Comment (ElizabethSamuel-MSFT) [inline docs/reference/requirement-sets/excel-api-online-requirement-set.md:44]:
```suggestion
- [Excel JavaScript API requirement sets](excel-api-requirement-sets.md)
```

**Review by ElizabethSamuel-MSFT — COMMENTED** [review]
Didn't notice this before, sorry.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[553d868](https://github.com/OfficeDev/office-js-docs-pr/commits/553d8688fa130c9e5b7b152f9596647d7738866a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-errors.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/excel/custom-functions-errors.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-errors?branch=pr-en-us-2180) |
[docs/excel/custom-functions-json-autogeneration.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/excel/custom-functions-json-autogeneration.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-json-autogeneration?branch=pr-en-us-2180) |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-overview?branch=pr-en-us-2180) |
[docs/excel/custom-functions-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/excel/custom-functions-requirement-sets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-requirement-sets?branch=pr-en-us-2180) |
[docs/excel/custom-functions-shared-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/excel/custom-functions-shared-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-shared-overview?branch=pr-en-us-2180) |
[docs/excel/custom-functions-web-reqs.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/excel/custom-functions-web-reqs.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-web-reqs?branch=pr-en-us-2180) |
[docs/excel/excel-add-ins-events.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/excel/excel-add-ins-events.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/excel-add-ins-events?branch=pr-en-us-2180) |
[docs/excel/excel-add-ins-ranges-advanced.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/excel/excel-add-ins-ranges-advanced.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/excel-add-ins-ranges-advanced?branch=pr-en-us-2180) |
[docs/excel/excel-add-ins-workbooks.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/excel/excel-add-ins-workbooks.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/excel-add-ins-workbooks?branch=pr-en-us-2180) |
[docs/reference/manifest/versionoverrides.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/reference/manifest/versionoverrides.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/manifest/versionoverrides?branch=pr-en-us-2180) |
[docs/reference/requirement-sets/dialog-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/reference/requirement-sets/dialog-api-requirement-sets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/dialog-api-requirement-sets?branch=pr-en-us-2180) |
[docs/reference/requirement-sets/excel-api-1-1-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/reference/requirement-sets/excel-api-1-1-requirement-set.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-api-1-1-requirement-set?branch=pr-en-us-2180) |
[docs/reference/requirement-sets/excel-api-1-10-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/reference/requirement-sets/excel-api-1-10-requirement-set.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-api-1-10-requirement-set?branch=pr-en-us-2180) |
[docs/reference/requirement-sets/excel-api-1-11-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/reference/requirement-sets/excel-api-1-11-requirement-set.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-api-1-11-requirement-set?branch=pr-en-us-2180) |
[docs/reference/requirement-sets/excel-api-1-12-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/reference/requirement-sets/excel-api-1-12-requirement-set.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-api-1-12-requirement-set?branch=pr-en-us-2180) |
[docs/reference/requirement-sets/excel-api-1-2-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/reference/requirement-sets/excel-api-1-2-requirement-set.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-api-1-2-requirement-set?branch=pr-en-us-2180) |
[docs/reference/requirement-sets/excel-api-1-3-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/reference/requirement-sets/excel-api-1-3-requirement-set.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-api-1-3-requirement-set?branch=pr-en-us-2180) |
[docs/reference/requirement-sets/excel-api-1-4-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/reference/requirement-sets/excel-api-1-4-requirement-set.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-api-1-4-requirement-set?branch=pr-en-us-2180) |
[docs/reference/requirement-sets/excel-api-1-5-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/reference/requirement-sets/excel-api-1-5-requirement-set.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-api-1-5-requirement-set?branch=pr-en-us-2180) |
[docs/reference/requirement-sets/excel-api-1-6-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/reference/requirement-sets/excel-api-1-6-requirement-set.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-api-1-6-requirement-set?branch=pr-en-us-2180) |
[docs/reference/requirement-sets/excel-api-1-7-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/reference/requirement-sets/excel-api-1-7-requirement-set.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-api-1-7-requirement-set?branch=pr-en-us-2180) |
[docs/reference/requirement-sets/excel-api-1-8-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/reference/requirement-sets/excel-api-1-8-requirement-set.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-api-1-8-requirement-set?branch=pr-en-us-2180) |
[docs/reference/requirement-sets/excel-api-1-9-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/reference/requirement-sets/excel-api-1-9-requirement-set.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-api-1-9-requirement-set?branch=pr-en-us-2180) |
[docs/reference/requirement-sets/excel-api-online-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/reference/requirement-sets/excel-api-online-requirement-set.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-api-online-requirement-set?branch=pr-en-us-2180) |
[docs/reference/requirement-sets/excel-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/reference/requirement-sets/excel-api-requirement-sets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets?branch=pr-en-us-2180) |
[docs/reference/requirement-sets/excel-preview-apis.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/reference/requirement-sets/excel-preview-apis.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-preview-apis?branch=pr-en-us-2180) |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-errors?branch=pr-en-us-2180) |
[docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-112Tables/docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial?branch=pr-en-us-2180) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C9%5C15%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202009152132223782-2180%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=W7MKuMWXOc6xtLgHHswnKDM6q863%2BFVDHAY%2Bwz1KcyY%3D&st=2020-09-15T21%3A27%3A58Z&se=2020-10-16T21%3A32%3A58Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by AlexJerabek** [comment]
@ElizabethSamuel-MSFT Fixed the description period in all the Excel req set articles and scrubbed the docs for unnecessary `./` references.

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Pull request merged**